src/cpip-pack

```diff
@@ -99,9 +99,9 @@ install_compilers()
         [[ $add ]] && [[ $sub ]] &&
             tmp_pkgs="$(jq_diff "$add" "$sub")"
 
-        # library packages stay, the rest are temporary
+        # library packages and pcre stay, the rest are temporary
         [[ $tmp_pkgs ]] &&
-            tmp_pkgs="$(echo "$tmp_pkgs" | jq -r '.[]' | grep -v "^lib")"
+            tmp_pkgs="$(echo "$tmp_pkgs" | jq -r '.[]' | grep -v "^lib\|^pcre")"
```

Comment:
do you know what the licences are on these? Is it ok for them to stay?

Reply:
looks like it's BSD

Reply:
> I approve 👍 and tested this locally
> but I wrote up the ticket so I'm biased 😄

Brilliant, great to know it's fixed the issue 👍

Reply:
I am not sure actually. Is BSD okay then?

Reply:
Just saw that most of the packages in eigen are BSD licensed. Nice 👍